README.md

```diff
@@ -210,6 +220,11 @@ You can run easily run your Stacks node with your own Bitcoin node by adding the
 
 Example: `./manage.sh -n mainnet -a up -f bitcoin` or `./manage.sh -n testnet -a up -f bitcoin`
 
+### Bitcoin node configuration
+
+In the `.env` file there is the variable `BITCOIN_BLOCKCHAIN_FOLDER`.
```

Comment:
No, not yet. 
This can be done for a next version. 
If you are "sharing" the data with another app you normally wouldn't want this app to delete it. But options are always good and it would be a nice to have feature.

Reply:
Update on this:
I added the reset functionality in a new commit.